src/utils/neverthrow.ts

```diff
@@ -0,0 +1,16 @@
+import { ResultAsync } from "neverthrow"
+
+/**
+ * This function is only used when integrating with third party libraries that
+ * expect a .catch() method on the returned promise. This should not be used in most
+ * control flows as it removes the benefits that neverthrow provides.
+ */
+const convertNeverThrowToPromise = <T, E>(x: ResultAsync<T, E>): Promise<T> =>
```

Comment:
does `x._unsafeUnwrap` work here?

Reply:
just tried, the below snippet works!

```ts
  const res = await x
  return res._unsafeUnwrap()

```

Reply:
done [097f1f4](https://github.com/isomerpages/isomercms-backend/pull/1044/commits/097f1f47d81f1c92cd5b915262eea832d0c26e7f)